test/spec/modules/sigmoidAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,67 @@
+import sigmoidAnalytic from 'modules/sigmoidAnalyticsAdapter';
+import { expect } from 'chai';
+let events = require('src/events');
+let adaptermanager = require('src/adaptermanager');
+let constants = require('src/constants.json');
+
+describe('sigmoid Prebid Analytic', function () {
```

Comment:
Your coverage is 70%. Please increase that to atleast 80%
To test and view use `gulp test-coverage` and `gulp view-coverage`